release/readme.html

```diff
@@ -2,248 +2,424 @@
+                        <li class="list-group-item" style="padding-top: 8px; padding-bottom: 8px;">RunPeriod</li>
+                        <li class="list-group-item" style="padding-top: 8px; padding-bottom: 8px;">Schedule:File</li>
+                        <li class="list-group-item" style="padding-top: 8px; padding-bottom: 8px;">Table:OneIndependentVariable</li>
+                        <li class="list-group-item" style="padding-top: 8px; padding-bottom: 8px;">Table:TwoIndependentVariables</li>
+                        <li class="list-group-item" style="padding-top: 8px; padding-bottom: 8px;">Table:MultiVariableLookup</li>
+                        <li class="list-group-item" style="padding-top: 8px; padding-bottom: 8px;">ThermalStorage:Ice:Detailed</li>
+                        <li class="list-group-item" style="padding-top: 8px; padding-bottom: 8px;">ZoneHVAC:EquipmentList</li>
+                    </ul>
+                </div>
+            </div>
+        </div>
+    </div>
+    
+    <div class="row">
+        <div class="col-md-12">
+            <a class="btn btn-primary margin-bottom-6" data-toggle="collapse" href="#collapseInputCheck" role="button"
+               aria-expanded="false"
+               aria-controls="collapseInputCheck">
+                Will my updated file definitely work?
+            </a>
+            <div class="collapse" id="collapseInputCheck">
+                <div class="card card-body">
+                    <h5 class="card-title">No, not definitely</h5>
+                    With every EnergyPlus release, there are new checks on user input. Even if your file ran in an
+                    earlier version, it may not run in the newer version after conversion due to newer checks on the
+                    validity of input.
+                </div>
+            </div>
+        </div>
+    </div>
+
+    <h2><a id="Multiple_Version_Conversions_51"></a>Multiple Version Conversions</h2>
+    <p>
+        The install includes the IDFVersionUpdater program and the necessary programs to convert from the previous
+        version (including .rvi and .mvi files) to the current version. For older versions, you will need to visit the
+        helpdesk and download the multiple conversion set of programs. The helpdesk downloads area:
+        <a href="http://energyplus.helpserve.com/Knowledgebase/List/Index/46/converting-older-version-files">
+            Converting Older Files</a>
+        will contain the necessary readmes and zip/tar files of programs to convert IDFs from V1.0 to
+        the current version.
+    </p>
+
+    <h2><a id="Documentation_55"></a>Documentation</h2>
+    <p>
+        The EnergyPlus documentation is included in pdf form in the installer. The pdfs are also available <a
+            href="https://energyplus.net/documentation">online</a>. For <strong>new users</strong> of EnergyPlus, a
+        guide called the EnergyPlus Essentials was developed and is available with the installed pdf documentation.
```

Comment:
Could this safely link to here?  https://energyplus.net/sites/all/modules/custom/nrel_custom/pdfs/pdfs_v9.1.0/EnergyPlusEssentials.pdf
Oh, it's got a version number in it. Maybe that could be changed to a more generic path in the future.

Reply:
Yeah, I wish it could link there dynamically, but I'm not sure we can count on it.